.travis.yml

```diff
@@ -1,14 +1,25 @@
 env:
   global:
     - REPO_DIR=onnx
-    - BUILD_COMMIT=0c070abb0c40fec649f81a73a75b0098662ec486
+    - BUILD_COMMIT=87eb5988a5cced69b946f3d3f2963a0e271940b9
     - PLAT=x86_64
     - PB_VERSION=2.6.1
     - UNICODE_WIDTH=32
```

Comment:
not being used any more... can be removed

Reply:
I thought you are saying the BUILD_COMMIT for Windows in `appveyor.yml`? Just removed.

Reply:
no I meant UNICODE_WIDTH

Reply:
Does default value of UNICODE_WIDTH equal to 32? I cannot find it in the document...